SwiftBaas-Example/Podfile.lock

```diff
@@ -1,6 +1,6 @@
 PODS:
   - Alamofire (4.7.2)
-  - CryptoSwift (0.9.0)
+  - CryptoSwift (0.10.0)
```

Comment:
Carthage only has 0.9.0 release so far. Maybe you ran again `pod update` instead of `pod install`?